lectures/greek_square.md

```diff
@@ -18,11 +18,21 @@ kernelspec:
 
 ## Introduction
 
-This lectures provides an  example of **invariant subspace** methods for analyzing linear difference equations. 
 
-These methods are applied throughout applied economic dynamics, for example, in this QuantEcon lecture {doc}`money financed government deficits and inflation <money_inflation>`
+This lecture can be viewed as a sequel to this QuantEcon lecture {doc}`eigenvalues and eigenvectors <eigen_I>` 
 
-Our approach in this lecture is to illustrate the method with an ancient example, one that ancient Greek mathematicians used to compute square roots of positive integers.
+It  provides an  example of how eigen vectors isolate  **invariant subspaces** that help construct and analyze solutions of  linear difference equations. 
```

Comment:
`eigen vectors` -> `eigenvectors`?

invariant subspaces should be italic and not bold